src/components/VmDetails/cards/OverviewCard/index.js

```diff
@@ -203,11 +203,20 @@ class OverviewCard extends React.Component {
 
     const showCloudInitCheckbox = isEditing && this.trackUpdates['name'] && !nameError && this.isCloudInitHostnameUpdate()
 
+    const poolId = vm.getIn(['pool', 'id'])
+    const isPoolVm = !!poolId
+    let pool = null
+    if (isPoolVm) {
+      pool = vms.getIn(['pools', poolId])
+    }
+    const isPoolAutomatic = pool && pool.get('type') === 'automatic'
+
     return (
       <BaseCard
         editMode={isEditing}
         editable={isEditable}
         editTooltip={`Edit ${vm.get('name')}`}
+        disableTooltip={isPoolVm && isPoolAutomatic ? msg.automaticPoolsNotEditable({ poolName: pool.get('name') }) : undefined}
```

Comment:
When I open a Pool VM as an admin user, the Overview card has a normal edit pencil button while the Details card has the disabled edit pencil button with the "automatic pool not editable" tooltip.  AFAICT they're supposed to both be disabled tooltipped pencil buttons.